infrastructure/tooling/src/build/tasks/client-shell.js

```diff
@@ -27,7 +27,7 @@ module.exports = ({ tasks, cmdOptions, credentials, baseDir, logsDir }) => {
       if (cmdOptions.staging || !cmdOptions.push) {
         // --snapshot will generate an unversioned snapshot release,
         // skipping all validations and without publishing any artifacts
-        goreleaserCmd = goreleaserCmd.push('--snapshot');
+        goreleaserCmd.push('--snapshot');
```

Comment:
this was assigning result of `Array.push()` (that returns int) to the same variable, which changes it from array to int. 
and later fails on the command execution step

Reply:
Ah thanks for catching! 🙏🏻